DQMOffline/Hcal/src/CaloTowersDQMClient.cc

```diff
@@ -132,10 +145,14 @@ int CaloTowersDQMClient::CaloTowersEndjob(const std::vector<MonitorElement*> &hc
          else
             phi_factor = 36.;
        }
-       if(ieta >= 0) ieta -= 1; // -41 -1, 0 40  - to bring back to histo num
 
-       cnorm = sumphi / phi_factor;
-       occupancy_vs_ieta->Fill(double(ieta), cnorm);
+      std::cout << "sumphi = " << sumphi << std::endl;
```

Comment:
Please remove this, std::cout is not allowed.

Reply:
Sorry about that, it's fixed now.